### PR TITLE
[API] SRI data in API

### LINKED ISSUE #190: SRI data in API
Current API doesn't provide SRI related data, should add.

### PR DESCRIPTION
**Pull request by rpidanny**
This fixes #190 

Adds `sri` field support and appends `sri` key in `assets`.

The documentation has been updated as well.

Closes #206
Closes #198

### CODE REVIEW

**Comment by Jakobud** [comment]
Looking forward to this!

**Comment by MattIPv4** [comment]
Dupe of #206 #198 ?

**Comment by Jakobud** [comment]
Any word on getting this merged in yet?

**Comment by MattIPv4** [comment]
@PeterDaveHello Could we get this reviewed & merged?

**Comment by MattIPv4** [comment]
cc #190

File: templates/api.html [hunk 1]
```diff
@@ -66,6 +66,7 @@ <h3>
 repository
 autoupdate
 author
+sri
```
Comment (PeterDaveHello) [inline templates/api.html]: SRI implementation here is under "assets", so adding it here may not be correct.

**Comment by JamieMagee** [comment]
@PeterDaveHello @MattIPv4 Is there any possibility to merge this? It would very much help with the work we're doing in renovatebot/renovate#5254

**Comment by MattIPv4** [comment]
I'm on a merging rampage currently, so yes, expect this to be live in at most a week :)

**Comment by JamieMagee** [comment]
@MattIPv4 thanks for getting this merged. When do you expect this change to be deployed?

**Comment by MattIPv4** [comment]
We're currently doing some work on cdnjs/cdnjs but once that is all happy we'll be pushing the latest SRI data to this repo and doing a deployment.

**Comment by BananaAcid** [comment]
Hi, what is the current state of affairs? Is it already usable, or close to it?

**Comment by MattIPv4** [comment]
Hi @BananaAcid, SRI data has been available on the API for quite a while now. You'll see it when making a request to `/libraries/<library>`.

**Pull request merged**